deploy-service/common/src/main/java/com/pinterest/deployservice/bean/EnvironBean.java

```diff
@@ -52,6 +54,7 @@
  * max_deploy_num      INT           NOT NULL,
  * max_deploy_day      INT           NOT NULL,
  * is_docker           TINYINT(1)    DEFAULT 0,
+ * max_parallel_deploy_percentage TINYINT(1)
```

Comment:
to be consistent, can we make it shorter? we have been user short names. something like max_parallel_pct